src/EditorFeatures/Core/Implementation/ExtractMethod/AbstractExtractMethodCommandHandler.cs

```diff
@@ -61,8 +61,8 @@ public VSCommanding.CommandState GetCommandState(ExtractMethodCommandArgs args)
                 return VSCommanding.CommandState.Unspecified;
             }
```

Comment:
We can simplify this also (get workspace from buffer) - no need to fetch a document just to get workspace instance.